base/timer.cpp

```diff
@@ -152,6 +152,11 @@ time_t StringToTimestamp(std::string const & s)
   return res;
 }
 
+time_t StringViewToTimestamp(std::string_view s)
+{
+  return base::StringToTimestamp(std::string{s});
```

Comment:
This is slow as a temporary string is created here. The original proposal was to move the current std::string implementation here, and fix it to work with std::string_view. And modify the original function with std::string to call this:
`return base::StringToTimestamp(std::string_view{s});`